tools/console/console.js

```diff
@@ -689,12 +689,13 @@ class Console extends ConsoleBase {
   // The caller should be OK with yielding --- it has to be in a Fiber and it can't be
   // anything that depends for correctness on not yielding.  You can also call nudge(false)
   // if you just want to update the spinner and not yield, but you should avoid this.
-  nudge(canYield) {
+  // TODO -> Check here
+  async nudge(canYield) {
```

Comment:
This is used in some places where code has to be synchronous (where ever canYield is false). We probably need to have both `nudge` and `nudgeAsync` to support both. This also would be nice for build plugins since making `nudge` async would be a breaking change for them.